android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java

```diff
@@ -155,7 +155,7 @@ public void setUpcomingSession() {
         Date current = new Date();
         for (Session sess : sortedSessions) {
             try {
-                Date start = DateUtils.getDate(sess.getStartsAt());
+                Date start = DateConversion.getDate(sess.getStartsAt());
```

Comment:
DateConverter would be more suitable. And it was the work of another PR. This issue is not related to this change. Please make only related changes to keep work trackable